src/conf_macros.m4

```diff
@@ -895,8 +895,9 @@ AC_ARG_ENABLE([files-domain],
               [AS_HELP_STRING([--enable-files-domain],
                               [If this feature is enabled, then SSSD always enables
                                a domain with id_provider=files even if the domain
-                               is not specified in the config file
-                              [default=no]])],
+                               is not specified in the config file [default=no]
+                               Please note this configure option is deprecated and
+                               will be removed in one of the next versions of SSSD.])],
```

Comment:
This phrase sounds strange to me. @justin-stephenson can you review it?

Reply:
It seems okay to me.